src/cart/README.env

```diff
@@ -140,3 +140,7 @@ This file lists the environment variables used in CaRT.
  . CRT_MRC_ENABLE
    When not set automatically disables MR caching via FI_MR_CACHE_MAX_COUNT=0
    envariable setting. Set to non 0 to re-enable MR caching in the provider.
+
+ . CRT_TEST_CONT
+   When set to 1, orterun does not automatically shut down other servers when
+   one server is shutdown. Used in cart internal testing.
```

Comment:
the way code is written, it will use --continuous flag even if CRT_TEST_CONT is set to 0 as the check is
"is not None" vs == 1

Reply:
No.  In cart_utils.py, line 334-336 is an explicit test for CRT_TEST_CONT equal to 1.  Any other value for the environmental variable will not set the --continuous flag.